docs/features/jsonSchema.md

```diff
@@ -19,32 +19,61 @@ The documentation below describes the new implementation.
 The new JSON Schema is structured to validate the YAML files that are loaded via JCasC.
 The structure and validation of the Schema is done based on the user-installed plugins.
 
-=== Version
+ ***Version***
```

Comment:
I do not agree with italics and bold styling.

Use headers:
```md
# Using JSON Schema

## Using JSON Schema

### Using JSON Schema

#### Using JSON Schema

##### Using JSON Schema

###### Using JSON Schema
```

# Using JSON Schema

## Using JSON Schema

### Using JSON Schema

#### Using JSON Schema

##### Using JSON Schema

###### Using JSON Schema